data.js

```diff
@@ -0,0 +1,51 @@
+const user = {
+  email: 'linsy.joyner@gmail.com',
+  username: 'linsydeanna',
+  password: 'supersecretpassword',
+  _id: '592c8029c1e6f9d97b6b146b',
+  trips: [
+    {
+      name: 'Europe 2017',
+      departureDate: '10/9/17',
+      returnDate: '10/16/17',
```

Comment:
Should we call it `startDate` and `endDate`? When I hear `departure` and `return`, I think of flights, which may be true in some cases, but not necessarily accurate for all trips.